app/app.js

```diff
@@ -24,6 +24,12 @@ const Menu = remote.Menu;
 const MenuItem = remote.MenuItem;
 
 const tinycolor = require('tinycolor2');
+const nunjucks = require('nunjucks');
```

Comment:
Done.

added another new special replacer 'dXx' which deletes any white spaces between hue name. Provided an example JSON file in my recent push.